consensus/utils.go

```diff
@@ -154,3 +201,38 @@ func isValidSignature(publicKey *bls.PublicKey, signature *bls.Signature, payloa
 	isValid, err := bls.VerifyAggregateSignatureSinglePayload([]*bls.PublicKey{publicKey}, signature, payload)
 	return err == nil && isValid
 }
+
+// This function uses integer math to verify if the provided stake amount represents a
+// super-majority 2f+1 Byzantine Quorum. First we need the following context:
+// - Assume N = total stake in the network
+// - Assume f = safe faulty stake in the network
```

Comment:
I'm not sure what "safe faulty" means here. Should we just say "faulty stake" or "malicious stake"?

Reply:
Removed the "safe" so it now says "faulty stake"